check.go

```diff
@@ -145,10 +145,10 @@ type CheckUpdateInput struct {
 	Name     string        `json:"name,omitempty"`
 	Enabled  *bool         `json:"enabled,omitempty"`
 	EnableOn *iso8601.Time `json:"enableOn,omitempty"`
-	Category interface{}   `json:"categoryId,omitempty"`
-	Level    interface{}   `json:"levelId,omitempty"`
-	Owner    interface{}   `json:"ownerId,omitempty"`
-	Filter   interface{}   `json:"filterId,omitempty"`
+	Category interface{}   `json:"categoryId"`
```

Comment:
So we'd have to change everything to an interface type?

Reply:
well no.  This was a change from your branch.  If we are good with this new scalar type we'd change everything to an `opslevel.ID`